src/appkit.cs

```diff
@@ -21412,6 +21543,43 @@ interface NSWorkspace : NSWorkspaceAccessibilityExtensions {
 		
 		[Export ("URLForApplicationToOpenURL:"), ThreadSafe]
 		NSUrl UrlForApplication (NSUrl url );
+
+		[Mac (12,0)]
+		[Export ("URLsForApplicationsWithBundleIdentifier:")]
+		NSUrl[] URLsForApplications (string bundleIdentifier);
```

Comment:
Verb + casing for acronym:

```suggestion
		NSUrl[] GetUrlsForApplications (string bundleIdentifier);
```